docs/examples/http2-download.c

```diff
@@ -203,6 +203,8 @@ int main(int argc, char **argv)
     curl_multi_add_handle(multi_handle, trans[i].easy);
   }
 
+  curl_multi_setopt(multi_handle, CURLMOPT_MAX_CONCURRENT_STREAMS, 120L);
```

Comment:
this is not needed in the boilerplate example. also I had intended my comments to be submitted all at once but I never clicked start review.. sorry about that!

Reply:
ok sure, shall remove this